src/Nethermind/Nethermind.Merge.Plugin.Test/EngineModuleTests.V3.cs

```diff
@@ -519,7 +519,7 @@ static void MarkAsUnprocessed(MergeTestBlockchain chain, int blockNumber)
 
         MergeTestBlockchain chain = await CreateBlockchain(releaseSpec: Cancun.Instance, mergeConfig: new MergeConfig()
         {
-            NewPayloadTimeout = TimeSpan.FromDays(1).TotalSeconds
+            NewPayloadTimeoutMs = 1000
```

Comment:
The original values is in days, not second. Probably something with randomly failing test.

Reply:
Thx, fixed